src/components/ui/timechart.tsx

```diff
@@ -1,4 +1,5 @@
 import React from "react";
+import EventCard from "./EventCard";
 
 const timechart = () => {
```

Comment:
as we are not using timechart, let's not make it timechart or name it that way, let's give proper name for the same ? Also I would prefer Named exports over default